.github/workflows/typos.yml

```
@@ -1,16 +1,13 @@
-name: Test GitHub Action
+name: Spelling
+
 on: [push, pull_request]
 
 jobs:
-  run:
-    name: Spell Check with Typos
+  spell-check-crate:
```

I prefer the previous name here.
```suggestion
  Spell Check with Typos:
```

---

:heavy_check_mark: